### PR TITLE
Share app feature

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #116 

Screenshots for the change: 

screenshot_2017-03-21-13-37-06-848_vn mbm phimp me

### CODE REVIEW

File: app/src/main/res/layout/settings.xml [hunk 1]
```diff
@@ -33,6 +33,15 @@
                     android:textColor="#ffffff"
                     android:textSize="20sp" />
 
+                <ImageView
+                    android:id="@+id/share_app"
+                    android:layout_width="wrap_content"
+                    android:layout_height="30dp"
+                    android:layout_alignParentRight="true"
+                    android:src="@drawable/share"
```
Comment (pa1pal) [inline app/src/main/res/layout/settings.xml]: Use vector assets for share image.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -635,4 +635,9 @@
     <!-- Settings View -->
     <string name="select_max_file_size">Select the Max File Size -</string>
 
+    <!--Share App strings-->
+    <string name="whatsapp_promo_msg_template">Hey, check out this awesome App! - %s</string>
```
Comment (pa1pal) [inline app/src/main/res/values/strings.xml]: The share is not particularly related to Whatsapp. Correct the name.

**Comment by pa1pal** [comment]
@rohanagarwal94 I agree with you. But right now the app is no near to release and adding this feature will be useless for so long. I am putting this on hold for now. Will definitely merge this in future.

**Comment by Abhi2424shek** [comment]
I think what you are saying is right. But he's worked it out on the other hand? Let's wait for others @pa1pal. Sharing the app which has bugs won't be good i guess.

**Comment by rohanagarwal94** [comment]
@pa1pal updated with the required changes.

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @rohanagarwal94 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/rohanagarwal94/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by mariobehling** [comment]
@rohanagarwal94 Thank you!

Please change the link to http://phimp.me. We will be using this site to promote the app.

There are conflicts now. Please help to resolve them. Appreciate your work. Keep it up!

**Comment by mariobehling** [comment]
@pa1pal Thank you for your concerns, but we are all aware that things are in progress. So, let's be more flexible. We need to consider things are in more detail once we are in production of course. Great to see you are working seriously on this. 

So, let's keep it more open for now and accept contributions getting this project up to speed and quality. Cheers.

**Comment by mariobehling** [comment]
@rohanagarwal94 Please resolve merge conflicts.

**Pull request merged**